### PR TITLE
page 404 new design

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Implemented new page 404 following design from issue #1968

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @lwrigh

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
Old 404:
old-404

New 404:
new-404

File: source/stylesheets/404.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    background-image: url(/images/404/404_bg.png);
+    background-size: contain;
+    width: 100%;
+}
+
+.circle {
+    -moz-border-radius: 100%;
+    -webkit-border-radius: 100%;
+    border-radius: 100% ;
+    border: 2px solid aqua;
```
Comment (bencurio) [inline source/stylesheets/404.css:18]: I think you should increase the border size a little bit:

```css
border: 10px solid aqua;
```

**Comment by fabiand** [comment]
/lgtm

looks great to me

**Review by nirs — APPROVED** [review]
Awesome!

**Comment by lwrigh** [comment]
This looks great to me!

**Comment by janoszen** [comment]
LGTM, would recommend taking a look at Bence's comment though.

**Comment by sandrobonazzola** [comment]
Added @bencurio suggestions. New screenshot:
Screenshot_2021-05-11 Page not found

**Comment by duck-rh** [comment]
It looks good but the colors are different. I preferred the deeper blue in the PNG. Reading the blue text also felt slightly easier. Anyway, it's still nice but maybe check we use the same set of colors in every oVirt places.

**Comment by sandrobonazzola** [comment]
@bencurio can you check @duck-rh comment?

**Comment by bencurio** [comment]
@sandrobonazzola Yes, of course! I agree with @duck-rh . My suggestion is the brand primary color `#0088CE` from oVirt bootstrap settings. 

Preview:
117824048-6a082f00-b26e-11eb-8caa-fde98a0eabef

Ref: https://github.com/oVirt/ovirt-site/blob/master/source/_sass/bootstrap-custom.sass#L29

**Comment by bencurio** [comment]
@sandrobonazzola Yes, of course! I agree with @duck-rh. My suggestion is the brand primary color `#0088CE` from oVirt bootstrap settings.

Preview:
image

Ref: https://github.com/oVirt/ovirt-site/blob/master/source/_sass/bootstrap-custom.sass#L29

**Comment by duck-rh** [comment]
Quack,

Thank for looking into it. The circle color is much better in your latest proposal IMHO. I would have made the background color slightly darker too.

Anyway, just providing feedback but I think that's a good improvement and we can merge the final result.

\\_o<

**Comment by sandrobonazzola** [comment]
Thanks everyone for the feedback and contribution. Adjusted the circle color as per @bencurio suggestion. I'll merge once CI passes. If we can make the background a bit darker we can do that in separate patch.

**Pull request merged**